docs/ConfigurationOptions/FolderConfiguration.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: Configuration options
+permalink: /configuration-options/
+nav_order: 5
+has_children: true
+---
+
+# Folder configuration
+
+The folder configuration in grate is now (since grate v 1.4) entirely configurable. That means you can 
+specify the folder structure of choice, and make grate work with your existing structure, rather than
+having to change the folder structure to suite how grate works. There is, however, a [default configuration](#default-folder-configuration),
+which might be a good starting point if you have no special requirements.
```

Comment:
Given the complexity (and some subtlety called out in your 'interprestion' note below, could I suggest we reword this as the default config "...provides the simplest starting point..." or similar to really try and dissuade people from shooting themselves in the foot with this unless they need it?

Reply:
Just to clarify, @wokket, would you prefer the wording:

There is, however, a [default configuration](#default-folder-configuration),
which provides the simplest starting point if you have no special requirements.

And, is the feature itself too complex, do you think? Could it be made easier to use?

Reply:
That wording sounds good to me.  I don't think it's _too_ complex, bit it's definitely _more_ complex than using the defaults.

I spend a bit of time in the MassTransit discord, and a surprisingly large amount of support issues boil down to "There was a knob, so I twiddled it, and now stuff's broken"... the solution to which is always "leave the knobs alone unless you **know** you need them".

I'm just trying to cut off similar issues here early by keeping people new to the tool on the simplest path possible.

Reply:
Just to pitch in, maybe we can offset @wokket's "issue" aswell by having the folderConfiguration in an advanced configurations. This way normal users are less likely to come across it.

Regarding the rest of the PR, it looks good to me!